lib/routers/establishment/projects.js

```diff
@@ -89,6 +95,28 @@ const submit = action => (req, res, next) => {
     .catch(next);
 };
 
+const loadRa = (req, res, next) => {
+  const { RetrospectiveAssessment } = req.models;
+  return Promise.resolve()
+    .then(() => RetrospectiveAssessment.query()
+      .select('id', 'status', 'status', 'createdAt', 'updatedAt')
+      .where({ projectId: req.project.id })
+      .orderBy('createdAt', 'desc')
+    )
+    .then(ras => {
+      const draft = ras && ras[0] && ras[0].status === 'draft' ? ras[0] : undefined;
+      // get most recent granted version.
+      const granted = ras.find(v => v.status === 'granted');
```

Comment:
Is it always the case that there is exactly one granted RA?

Reply:
👍